File: interview/clever/quiz.rb
require 'net/https'
uri = URI('https://api.clever.com/v1.1/students?count=true')

http = Net::HTTP.new(uri.host, uri.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_PEER

request = Net::HTTP::Get.new(uri.request_uri)
request.add_field 'Authorization', 'Bearer DEMO_TOKEN'

response = http.request(request)
puts response.body

File: interview/clever/quiz.js
var https = require('https');

var options = {
  method: 'GET',
  hostname: 'api.clever.com',
  path: '/v1.1/sections?count=true',
  headers: {
    Authorization: 'Bearer DEMO_TOKEN'
  }
};

var request = https.request(
  options, 
  function(response) { 
    response.setEncoding('utf8');
    response.on('data',
      function (chunk) {
        console.log(chunk);
      }
    )
  }
);

request.end();
